lib/mayaUsd/fileio/translators/translatorCamera.cpp

```diff
@@ -470,9 +471,17 @@ bool _ReadToCamera(
     }
 
     usdAttr = usdCamera.GetFStopAttr();
-    plugName = _tokens->MayaCameraAttrNameFStop;
-    if (!_TranslateUsdAttributeToPlug(usdAttr, cameraFn, plugName, args, context)) {
-        return false;
+    if (usdAttr.IsAuthored()) {
```

Comment:
Is the IsAuthored() test enough to catch all cases where fStop is zero? Looking at the docs it says it'll return true if there any any opinions on any layer, so if there is an opinion which sets it zero I think we'll get into this code block which represents a non-zero fStop.

Reply:
Yes, thought I added a **FIXME** in there to that effect, but I guess not.

The reason I didn't tackle it was that _really_ checking for zero could incur iteration over all time samples, and I thought that part would best best be left to phase 2 if Maya import/export had to write a custom property for keyframes.

If we're not going to be worrying about time sampled data for now (which for our case isn't currently relevant) then I'll go ahead and add the read-value.